examples/contour-dev/README.md

```diff
@@ -0,0 +1,29 @@
+# Contour-dev Installation
+
+This is an installation guide to configure Contour in a standalone Daemonset for easy trialling or development.
```

Comment:
Why? Can we use a deployment rather than a DS?

Reply:
Cargo culting. I'll change it to a Deployment.